endpoints-control/src/main/java/com/google/api/control/Client.java

```diff
@@ -390,6 +401,15 @@ private void flushAndScheduleReports() {
             String.format("direct send of a report request failed because of %s", e));
       }
     }
+    if (flushed.length > 0) {
+      reportStopwatch.reset().start();
+    } else if (reportStopwatch.elapsed(TimeUnit.SECONDS) > MAX_IDLE_TIME_SECONDS) {
+      log.log(Level.INFO,
+          String.format(
+              "Shutting down after no reports in the last %d seconds.", MAX_IDLE_TIME_SECONDS));
```

Comment:
Shouldn't this actually be the stopwatch's elapsed time, since it may be greater than the constant?

Reply:
I don't think it matters. They will be within 1-2 seconds of each other.